### DISCUSSION TITLE
use vue-i18n and config.globalProperties.$t at the same time, my own $t function will be coverd

### DISCUSSION THREAD

**Discussion by luolei1994** [intlify/vue-i18n#942] [Q&A]
use vue-i18n and config.globalProperties.$t at the same time,  my own $t function will be coverd by vue-i18n 's $t.but i try vue-router,it not.

in vue2 i use vue-i18n's $t and Vue.prototype.$t to handle the key.
![image](https://user-images.githubusercontent.com/22974278/156490252-08a0883f-a8ca-4122-9c4e-bc4d458c73be.png)

but, in vue3, it can't work.

**Comment by LasyIsLazy** [comment]
You should use `app.config.globalProperties` to set global properties not `prototype` in Vue3.
